packages/mdc-top-app-bar/README.md

```diff
@@ -146,11 +163,14 @@ Mixin | Description
 `mdc-top-app-bar-fill-color-accessible($color)` | Sets the fill color of the top app bar and automatically sets a high-contrast ink color.
 `mdc-top-app-bar-short-border-radius($border-radius)` | Sets the `border-bottom-radius` property on the action item side. Used only for the short top app bar when collapsed.
 
-### `MDCTopAppBar`
+## `MDCTopAppBar` Properties and Methods
+
+### `MDCTopAppBarComponent`
```

Comment:
This third-level heading shouldn't exist anymore (and `MDCTopAppBarComponent` doesn't exist)